Configuration/Generator/python/WM_14TeV_TuneCUETP8M1_cfi.py

```diff
@@ -15,7 +15,7 @@
         processParameters = cms.vstring(
             'WeakSingleBoson:ffbar2W = on',
             '24:onMode = off',
-            '24:onIfAny = 12 13',
+            '24:onIfAny = 13 14',
             ),
         parameterSets = cms.vstring('pythia8CommonSettings',
                                     'pythia8CUEP8M1Settings',
```

Comment:
Are you going to request this sample as well? If so, should we also update to TuneCP5?

Reply:
I'm really sorry --I have issue w/ the email filtering (!!!) :( and I've just seen all these comments (!!!)

I simply patch this config because I thought it was wrong, I can update --for consistency-- this config as well, thanks for the comment